### PR TITLE
1_html_vBrusentsov

### PR DESCRIPTION
**Pull request by vBrusentsov**

### CODE REVIEW

**Comment by vadym-zinchenko-moc** [comment]
Screenshot 2023-10-21 at 14 04 22

 - Your layout isn't correct because you has only one paragraph. 
 - You need add links (your github and  social media instead [Links to Socials Nets]  and GitHub)
 -

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.main__content {
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 300;
+}
+
+p a {
+  color: #EF4934;
+}
```
Comment (vadym-zinchenko-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css:17]: Better use classes instead tags for styling.
And read pls about methodology BEM and use this pls.

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <title>About</title>
+    </head>
+    <body>
+        <header></header>
+        <main>
+            <h1>About me</h1>
+            <p class="main__content">Hi! My name is Vladyslav Brusentsov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system. 
+                This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023.</a>
+                This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio. 
+                You can contact me via [Links to Socials Nets] and/or check out my GitHub.
```
Comment (vadym-zinchenko-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html:16]: You have to change layout like on design

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html (hunk 2 above)
Comment (vBrusentsov) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html]: Done

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css (hunk 1 above)
Comment (vBrusentsov) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css:17]: Okay. Thank you I did it)

Comment (vadym-zinchenko-moc) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css:17]: You still use tag a for styling) pls add class to this tag. 
Something like this:
.link {
  color: #EF4934;
}

**Review by vadym-zinchenko-moc — CHANGES_REQUESTED** [review]

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css (hunk 1 above)
Comment (vBrusentsov) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/stylesheet.css:17]: Okay I did it, thank you

**Pull request closed without merging**